_includes/payment-resource.md

```diff
@@ -35,7 +35,7 @@ Content-Type: application/json
         "state": "Ready",
         "operation": "Purchase",
         "intent": "Authorization",
-        "currency": "NOK",
+        "currency": "SEK",
```

Comment:
Is it correct to always have `SEK` here or should `currency` become a parameter (that can default to `SEK`)?